ports/esp32/mphalport.h

```diff
@@ -52,6 +52,7 @@
 
 extern TaskHandle_t mp_main_task_handle;
 
+extern int mp_interrupt_char;
```

Comment:
I think it's better to include `shared/runtime/interrupt_char.h` rather than declare this variable.